test_mavros/include/tests/offboard_control.h

```diff
@@ -0,0 +1,470 @@
+/**
+ * @brief Offboard control test
+ * @file offboard_control.h
+ * @author Nuno Marques <n.marques21@hotmail.com>
+ * @author Andre Nguyen <andre-phu-van.nguyen@polymtl.ca>
+ *
+ * @addtogroup sitl_test
+ * @{
+ */
+/*
+ * Copyright 2015 Nuno Marques, Andre Nguyen.
+ *
+ * This file is part of the mavros package and subject to the license terms
+ * in the top-level LICENSE file of the mavros repository.
+ * https://github.com/mavlink/mavros/tree/master/LICENSE.md
+ */
+
+#include <mavros/mavros.h>
```

Comment:
Well this one doesn't so I can remove. For other test that regard on timestamps, etc., probably they will.